src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+body,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  --active-color: #00acdc;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.header__logo {
+  margin: 10px 0;
+  height: 40px;
+}
+
+.nav__menu {
+  display: flex;
+  height: 100%;
+}
+
+.nav__button {
+  list-style: none;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-style: normal;
```

Comment:
`font-family` should be insude `html` selector
`font-style` by default normal redudant